example/client/src/di.config.ts

```diff
@@ -20,11 +20,13 @@ import { GLSPDiagramServer } from "@eclipse-glsp/client";
 import { Container } from "inversify";
 import { ConsoleLogger, LogLevel, TYPES } from "sprotty";
 
+import { KeepAliveActionHandler } from "./keep-alive-action-handler";
 
 export default function createContainer(): Container {
     const container = createWorkflowDiagramContainer('sprotty');
     container.bind(TYPES.ModelSource).to(GLSPDiagramServer).inSingletonScope();
     container.rebind(TYPES.ILogger).to(ConsoleLogger).inSingletonScope();
     container.rebind(TYPES.LogLevel).toConstantValue(LogLevel.warn);
+    container.bind(TYPES.IActionHandlerInitializer).to(KeepAliveActionHandler).inSingletonScope();
```

Comment:
The `IActionHandlerInitializer` API of sprotty is deprecated. Please use `configureActionHandler` instead.

`
const context= {bind:container.bind,isBound:container.isBound}
configureActionHandler(context,KeepAliveAction.KIND,KeepAliveActionHandler);
`

Reply:
```
Uncaught TypeError: Cannot read property 'hasKey' of undefined
    at Object../node_modules/inversify/lib/container/container.js.Container.isBound (container.js:181)
    at Object.configureActionHandler (action-handler.js:82)
    at Object.createContainer [as default] (di.config.js:30)
    at Object../lib/main.js (main.js:29)
    at __webpack_require__ (bootstrap:19)
    at Object.0 (module.js:22)
    at __webpack_require__ (bootstrap:19)
    at bootstrap:83
    at bootstrap:83
```
When I try this, the _bindingDictionary of the inversify container is undefined when registering the keep alive action. 
Have you seen this error before and know the solution?

Reply:
The following seems to work (I compared the usage with other GLSP code and noticed that configureActionHandler was always used from ContainerModules)

```
    const keepAliveModule = new ContainerModule((bind, _unbind, isBound) => {
        configureActionHandler({ bind, isBound }, KeepAliveAction.KIND, KeepAliveActionHandler);
    });
    container.load(keepAliveModule)
```